docs/k8s-helm.md

```diff
@@ -33,7 +35,8 @@ NAME                        	CHART VERSION	APP VERSION	DESCRIPTION
 timeplus/timeplus-enterprise	v4.0.10      	2.5.11     	Helm chart for deploying a cluster of Timeplus ...
 timeplus/timeplus-enterprise	v3.0.7       	2.4.23     	Helm chart for deploying a cluster of Timeplus ...
 ```
-Please choose the latest `CHART VERSION`. Staring from v3.0.0 chart version, the `APP VERSION` is the same version as [Timeplus Enterprise](/enterprise-releases).
+
+Staring from v3.0.0 chart version, the `APP VERSION` is the same version as [Timeplus Enterprise](/enterprise-releases). Prior to v3.0.0 chart version, the `APP VERSION` is the same version as the timeplusd component.
```

Comment:
> Please choose the latest `CHART VERSION`

I think this "advice" is not necessary. Always use latest version should be the default mindset for most of the people. If people don't use latest version, they must have their own reasons (e.g. they want a stable build).

Reply:
Yes, good to remove 'choose the latest ..'